src/test/groovy/com/athaydes/spockframework/report/internal/HtmlReportCreatorSpec.groovy

```diff
@@ -273,4 +298,18 @@ class HtmlReportCreatorSpec extends ReportSpec {
         }
     }
 
+    @Category( SpockReportExtension )
+    class ShowCodeBlocksEnabled {
+        static final htmlReportCreator = new HtmlReportCreator()
+        SpecInfoListener createListener() {
+            setShowCodeBlocks( true )
+            configReportCreator htmlReportCreator
+            new SpecInfoListener( htmlReportCreator )
+        }
+    }
+
+    @Category( SpockReportExtension )
+    class ShowCodeBlocksDisabled {
```

Comment:
Ah... I withdraw my comment :) I didn't realize we have two sources of blocks, and that's why you split up the methods into the code-based ones and the text-based ones. It's good as it is.